src/semidiscretization.jl

```diff
@@ -46,7 +46,10 @@ end
                        boundary_conditions=boundary_condition_periodic,
                        RealT=real(solver),
                        uEltype=RealT,
-                       initial_cache=(tmp1 = Array{RealT}(undef, nnodes(mesh)),))
+                       initial_cache = (tmp1 = Array{RealT}(undef, nnodes(mesh)),
+                                        tmp_partitioned = allocate_coefficients(mesh,
+                                                                                     equations,
+                                                                                     solver)))
```

Comment:
```suggestion
                                        tmp_partitioned = allocate_coefficients(mesh, equations, solver)))
```
Just some aesthetics.